### PR TITLE
7.3 LPS-102734

### PR DESCRIPTION
**Pull request by mwilliams2014**

### CODE REVIEW

File: en/user/articles/100-web-experience-management/03-building-a-site/03-adding-pages-to-sites/03-content-pages/03-content-page-interface.markdown [hunk 1]
```diff
@@ -92,6 +92,11 @@ each piece of content used on the page:
 - *Permissions:* Update the web content's permissions
 - *View Usages:* See where the web content is used on the site
 
+| **Note:** To edit inline content, you must have the "Update Page Content" 
+| permission assigned to your Role. By default this permission is included for 
+| the BLANK role. 
+<!--What Role is the permission included with by default?-->
```
Comment (ruben-pulido) [inline en/user/articles/100-web-experience-management/03-building-a-site/03-adding-pages-to-sites/03-content-pages/03-content-page-interface.markdown]: This permission is not included  by default in any role.

File: en/user/articles/100-web-experience-management/03-building-a-site/03-adding-pages-to-sites/03-content-pages/08-creating-new-content-from-inline-content.markdown [hunk 2]
```diff
@@ -14,6 +14,13 @@ content, you can select the inline content and save it as new web content that
 you can reuse across multiple pages. To create new web content from inline 
 content, you must first select the content you want to include.
 
+| **Note:** To edit inline content, you must have the "Update Page Content" 
+| permission assigned to your Role. By default this permission is included for 
+| the BLANK role. 
+<!--What Role is the permission included with by default?-->
```
Comment (ruben-pulido) [inline en/user/articles/100-web-experience-management/03-building-a-site/03-adding-pages-to-sites/03-content-pages/08-creating-new-content-from-inline-content.markdown]: This permission is not included by default in any role.

**Comment by ruben-pulido** [comment]
Hi @mwilliams2014 
- "Creating new content from page" is part of this epic https://issues.liferay.com/browse/LPS-103803, whose documentation should not be published yet, since the epic is still under development and the functionality is currently deactivated.
- We can also include documentation covering the functionality implemented in https://issues.liferay.com/browse/LPS-103454, which allows a user updating content mapped to a page without the need for the "Update Inline Content" permission

**Comment by mwilliams2014** [comment]
Thanks @ruben-pulido ! I'll make the updates you suggested to the draft.

**Comment by mwilliams2014** [comment]
Closing. Info included in PR to Liferay Learn: https://github.com/jrhoun/liferay-learn/pull/188

**Pull request closed without merging**